### DISCUSSION TITLE
Is it possible to have multiple tsparticles (like links and stars) on the same screen?

### DISCUSSION THREAD

**Discussion by heisenbuggs** [matteobruni/tsparticles#5256] [Q&A] [answered]
I tried to have multiple tsparticles onscreen but only one of them (the latest one) renders each time. I want to show two particles on my screen (links and stars). Please let me know how is it possible?

**Comment by matteobruni** [comment, marked as answer]
Use different ids

**Reply by heisenbuggs** [reply]
Thanks for the help! It did work 😄
